docs/reference/builder.md

```diff
@@ -1754,6 +1754,6 @@ FROM ubuntu
 RUN echo moo > oink
 # Will output something like ===> 695d7793cbe4
 
-# You᾿ll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
+# You'll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
```

Comment:
Curly quote is not incorrect here, but for CLI examples / comment, I'm ok to change it (http://practicaltypography.com/straight-and-curly-quotes.html)